cypress/e2e/homepage-spec.cy.js

```diff
@@ -1,5 +1,41 @@
-describe('template spec', () => {
-  it('passes', () => {
-    cy.visit('https://example.cypress.io')
-  })
-})
\ No newline at end of file
+describe('homepage search page spec', () => {
+  beforeEach(() => {
+    cy.intercept('GET', 'http://localhost:3000/sockjs-node/info?t=*', {
+      statusCode: 200,
+      fixture: 'websocket',
+    });
```

Comment:
What's this for?

Reply:
It's just a weird fetch that was showing up in my cypress tests. No clue why it's there, but it was there so I stubbed it.